bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v2/form/FileInputImplV2.java

```diff
@@ -57,7 +57,7 @@ public String getDragDropText() {
     @Override
     public @NotNull Map<String, Object> getProperties() {
         Map<String, Object> customProperties = super.getProperties();
-        customProperties.put(ReservedProperties.PN_DRAG_DROP_TEXT, getDragDropText());
+        customProperties.put("fd:" + ReservedProperties.PN_DRAG_DROP_TEXT, getDragDropText());
```

Comment:
Ideally, the property name should also have fd: prefix in this, it was a miss in the earlier review, for the new v3 component, let's persist the property also as fd:dragDropText